sdks/python/apache_beam/runners/dataflow/ptransform_overrides.py

```diff
@@ -236,7 +236,10 @@ def enter_composite_transform(self, transform_node):
         self.visit_transform(transform_node)
 
       def visit_transform(self, transform_node):
-        if [o for o in self.outputs if o in transform_node.inputs]:
+        # Internal consumers of the outputs we're overriding are expected.
```

Comment:
Why did we have to add this explicit fail ? I don't think we add such visitors for other transforms that do not produce output. Usually succeeding transform just ends up being an no-op.

Reply:
This is because BQ FIle loads returns a series of pcollections (e.g. files written, jobs started) - and if the downstream transforms depend on these outputs, they will not work properly when replacing WriteToBQ with BQSink, so we error out.